docs/UserGuide.md

```diff
@@ -78,13 +73,97 @@ Fig 1. Home tab
 {:.caption}
 
 #### 2.2.2 Inventory & Statistics Tab
+
 Displays [inventory](#32-inventory) and [statistics](#35-statistics).
 
 ![Inventory & Statistics tab](images/product-screenshots/general/BarChart.png)
 
 Fig 2. Inventory & Statistics tab
 {:.caption}
 
+### 2.3 Creating your first order
+
+This is a step-by-step guide to creating your first order. If you have not installed BobTheBistroBoss, you
+may do so by following the instructions [here](#21-installation).
+
+For this guide, we will simulate the following scenario:
+
+John Doe wishes to place two orders, one on 10 October 2021 and the other on 11 October 2021. Both of these orders are
```

Comment:
Can you make this more user-centric. Something like your regular customer John Doe....